pkg/idutil/id.go

```diff
@@ -43,7 +43,7 @@ const (
 //
 // It increases suffix to generate the next id.
 // The count field may overflow to timestamp field, which is intentional.
-// It helps to extend the event window to 2^56. This doesn't break that
+// It helps to extend the event window to 2^48. This doesn't break that
```

Comment:
For reference: In the original PR https://github.com/etcd-io/etcd/pull/1952, the prefix was 1 byte, so it's indeed 7 bytes (56bit). The prefix was changed to 2 bytes later, but the doc/comment wasn't updated accordingly.

https://github.com/etcd-io/etcd/pull/1952

thx